package.json

```diff
@@ -115,15 +108,7 @@
     "chai": "^4.3.4",
     "enzyme": "^3.4.1",
     "esbuild": "^0.19.8",
-    "eslint": "^8.23.0",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jsx-a11y": "^6.6.1",
-    "eslint-plugin-react": "^7.31.8",
-    "eslint-plugin-react-hooks": "^4.6.0",
     "pixi.js": "^6.5.2",
-    "prettier": "^2.5.1",
     "react": "^17.0.2",
```

Comment:
To me, this is the biggest benefit. We get a single dependency with reasonable/strict linting rules that takes care of all (and more) than what these 8 packages were doing for us before.